### ISSUE TITLE
Reading: UDALM: Unsupervised Domain Adaptation through Language Modeling

### ISSUE THREAD

**Issue by a1da4** [a1da4/paper-survey#160]
## 0. Paper
Accepted for NAACL2021 [arxiv](https://arxiv.org/abs/2104.07078)

## 1. What is it?
They propose a new unsupervised domain adaptation method using BERT.

## 2. What is amazing compared to previous works?
* They propose a new unsupervised domain adaptation method based on multitask learning.
* Their method achieved state-of-the-art performances for the Amazon review dataset.

## 3. Where is the key to technologies and techniques?
Their key idea is minimizing task-specific loss (source domain) and musked language modeling loss (target domain) simultaneously.
 
![スクリーンショット 2021-04-16 23 06 57](https://user-images.githubusercontent.com/45454055/115036402-723bab80-9f08-11eb-8fa5-9c6077f75b22.png)
![スクリーンショット 2021-04-16 23 27 49](https://user-images.githubusercontent.com/45454055/115039375-5e457900-9f0b-11eb-8fce-601b38fb3c89.png)

lambda is a hyperparameter that is decided by the ratio of labeled source data (n) over the sum of labeled source (n) and unlabeled target data (m)
![スクリーンショット 2021-04-16 23 28 30](https://user-images.githubusercontent.com/45454055/115039495-75846680-9f0b-11eb-91e5-7ed37edeea4a.png)

## 4. How did evaluate it?
Amazon reviews in multi-domain sentiment dataset
* B: Books
* D: DVDs
* E: Electronics
* K: Kitchen appliances

They compared their model UDALM and:
* Previous methods
  * R-PERL: pivot based
  * DAAT: domain adversarial training
  * p+CFd: self-train XLM-R based
* Ablation methods
  * SO BERT: fine-tuning only source domain 
  * DPT BERT: pre-training and fine-tuning with target domain
  * DAT BERT: domain adversarial training with BERT
![スクリーンショット 2021-04-16 23 36 53](https://user-images.githubusercontent.com/45454055/115040671-a2854900-9f0c-11eb-818e-8a8e19fb6a4b.png)

* vs previous methods: UDALM > p+CFd > DAAT > R-PERL
* vs ablation methods: UDALM > DPT > SO > DAT

![スクリーンショット 2021-04-16 23 30 12](https://user-images.githubusercontent.com/45454055/115039752-b2e8f400-9f0b-11eb-8b5d-a4a06808c6f1.png)

### 5.1 Different adaptation scenarios in unlabeled target dataset
![スクリーンショット 2021-04-16 23 40 56](https://user-images.githubusercontent.com/45454055/115041195-335c2480-9f0d-11eb-8814-75eab3ba0844.png)

### 5.2 Limitation of DAT
SO, DPT, and UDALM clearly separate positive and negative [CLS] vectors in the source and target datasets.
However, DAT provides **distortion in the semantic space**, which is reflected in the experimental results.
![スクリーンショット 2021-04-16 23 43 44](https://user-images.githubusercontent.com/45454055/115041541-96e65200-9f0d-11eb-9c4f-ac862d017961.png)

## 6. Which paper should read next?
* this paper is inspired by [Chronopoulou2019](https://www.aclweb.org/anthology/N19-1213/)
